### PR TITLE
Support NPU kernel sum op

### PR DESCRIPTION
**Pull request by oyxuan-11**
### PR types
New features

### PR changes
OPs

### Describe
【NPU】Support npu op sum

单测通过截图
image

调用NPU截图
image

### CODE REVIEW

File: paddle/fluid/operators/sum_op_npu.cc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    auto runner = NpuOpRunner("Add", {*x[0], *x[1]}, {*out}, {});
+
+    runner.Run(stream);
+    for (int i = 2; i < n; i++) {
+        runner = NpuOpRunner("Add", {*out, *x[i]}, {*out}, {});
+        runner.Run(stream);
+    }
+
+    std::cout << "After Npu OP Runner "<< std::endl;
```
Comment (liym27) [inline paddle/fluid/operators/sum_op_npu.cc]: Delete line 55

**Review by liym27 — COMMENTED** [review]
LGTM for the rest

File: paddle/fluid/operators/sum_op_npu.cc (hunk 1 above)
Comment (oyxuan-11) [reply paddle/fluid/operators/sum_op_npu.cc]: fixed

**Review by liym27 — APPROVED** [review]
LGTM

**Pull request merged**